mvcc/backend/backend.go

```diff
@@ -358,7 +358,14 @@ func (b *backend) defrag() error {
 
 	b.batchTx.tx = nil
 
-	tmpdb, err := bolt.Open(b.db.Path()+".tmp", 0600, boltOpenOptions)
+	tmpdbPath := b.db.Path() + ".tmp"
+	if _, err := os.Stat(tmpdbPath); err == nil {
```

Comment:
Can we log when delete fails?

e.g.

```go
// expect err == nil when the file exists and is deleted
// expect os.IsNotExist if the file does not exist
if err := os.Remove(p); err != nil && !os.IsNotExist(err) {
  b.lg.Warn("failed to...")
}
```

Reply:
Good catch. I'll also return an error in this case.